src/codec/video/frame.rs

```diff
@@ -490,6 +492,36 @@ impl VideoFrame {
     pub(crate) fn as_ptr(&self) -> *const c_void {
         self.ptr
     }
+
+    /// Try to make this frame mutable. Returns VideoFrameMut if it can be made
+    /// into mutable without copying the data, otherwise returns VideoFrame.
+    pub fn try_into_mut(self) -> Result<VideoFrameMut, VideoFrame> {
+        let res = unsafe { ffw_frame_is_writable(self.ptr) };
+        return if res > 0 {
+            Ok(self.into_mut())
+        } else {
+            Err(self)
+        };
```

Comment:
Just to make Clippy happy :)

```suggestion
        if res > 0 {
            Ok(self.into_mut())
        } else {
            Err(self)
        }
```